### PR TITLE
init the websockets

### PR DESCRIPTION
**Pull request by quirkles**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,6 +7,7 @@
     "build": "next build",
     "start": "next start",
     "lint": "next lint",
+    "graphql-gen": "next lint",
```
Comment (jwellwood) [inline package.json]: Is this intentional?

File: src/app/home/page.tsx [hunk 2]
```diff
@@ -47,22 +48,32 @@ interface HomeLoggedInProps{
...
             }
         })
         }
-    }, [username, userId, puzzleRating]);
+    }, [username, lichessUserId, puzzleRating]);
+    useEffect(() => {
+        if(userId) {
+            eventSocketService.notifyLogin(userId)
+            return eventSocketService.on(
+                'GameStart',
```
Comment (jwellwood) [inline src/app/home/page.tsx]: What's this?

File: package.json (hunk 1 above)
Comment (quirkles) [reply package.json]: no, removed

File: src/app/home/page.tsx (hunk 2 above)
Comment (quirkles) [reply src/app/home/page.tsx]: setting up the websocket, when the game starts, we should redirect to the game, this is mostly just testing and playing around for now

**Review by jwellwood — APPROVED** [review]

**Pull request merged**